scripts/language_model/xlnet_qa_evaluate.py

```diff
@@ -0,0 +1,152 @@
+# Copyright 2018 The Google AI Language Team Authors, Allenai and DMLC.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""XLNet SQuAD evaluate."""
```

Comment:
Should this be added after or as part of #1012?

Reply:
Hi, I planned to add it after [#1012](https://github.com/dmlc/gluon-nlp/pull/1012), It is the same with [#1012](https://github.com/dmlc/gluon-nlp/pull/1012).

Reply:
Would it make sense to add the Glue part independently of #1012. That way it's not blocked until #1012 and review would be simplified.
You can also consider directly merging the "refactored" squad API in #1012.